package/src/components/AttachmentPicker/AttachmentPicker.tsx

```diff
@@ -248,13 +248,18 @@ export const AttachmentPicker = React.forwardRef(
     }, [selectedPicker]);
 
     useEffect(() => {
-      if (Platform.OS === 'ios') {
-        Keyboard.addListener('keyboardWillShow', hideAttachmentPicker);
-      } else {
-        Keyboard.addListener('keyboardDidShow', hideAttachmentPicker);
-      }
+      const keyboardListener =
```

Comment:
`keyboardListener` -> `keyboardSubscription` is a little bit clearer variable name in my view